frontend/appflowy_flutter/lib/plugins/database/tab_bar/mobile/mobile_tab_bar_header.dart

```diff
@@ -51,131 +56,80 @@ class _MobileTabBarHeaderState extends State<MobileTabBarHeader> {
-    super.didUpdateWidget(oldWidget);
-    if (widget.isSelected != oldWidget.isSelected) {
-      statesController.update(MaterialState.selected, widget.isSelected);
-    }
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    return TextButton(
-      statesController: statesController,
-      style: ButtonStyle(
-        padding: const MaterialStatePropertyAll(
-          EdgeInsets.symmetric(horizontal: 12, vertical: 7),
-        ),
-        maximumSize: MaterialStateProperty.resolveWith((states) {
-          if (states.contains(MaterialState.selected)) {
-            return const Size(150, 48);
-          }
-          return const Size(120, 48);
-        }),
-        minimumSize: const MaterialStatePropertyAll(Size(48, 0)),
-        shape: const MaterialStatePropertyAll(
-          RoundedRectangleBorder(
-            borderRadius: BorderRadius.all(Radius.circular(12)),
-          ),
-        ),
-        backgroundColor: MaterialStateProperty.resolveWith((states) {
-          if (states.contains(MaterialState.selected)) {
-            return const Color(0x0F212729);
-          }
-          return Colors.transparent;
-        }),
-        overlayColor: MaterialStateProperty.resolveWith((states) {
-          if (states.contains(MaterialState.selected)) {
-            return Colors.transparent;
-          }
-          return Theme.of(context).colorScheme.secondary;
-        }),
-      ),
-      child: Row(
-        mainAxisSize: MainAxisSize.min,
-        children: [
-          _buildViewIconButton(context, widget.tabBar.view),
-          const HSpace(6),
-          Flexible(
-            child: FlowyText(
-              widget.tabBar.view.name,
-              fontSize: 14,
-              fontWeight: widget.isSelected ? FontWeight.w500 : FontWeight.w400,
-              overflow: TextOverflow.ellipsis,
+            backgroundColor: const MaterialStatePropertyAll(Color(0x0F212729)),
+            overlayColor: MaterialStatePropertyAll(
+              Theme.of(context).colorScheme.secondary,
             ),
           ),
-        ],
-      ),
-      onPressed: () {
-        if (!widget.isSelected) {
-          context
-              .read<DatabaseTabBarBloc>()
-              .add(DatabaseTabBarEvent.selectView(widget.tabBar.viewId));
-        }
+          child: Row(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              _buildViewIconButton(context, tabBar.view),
+              const HSpace(6),
+              Flexible(
+                child: FlowyText.medium(
+                  tabBar.view.name,
+                  fontSize: 13,
+                  overflow: TextOverflow.ellipsis,
+                ),
+              ),
+              const HSpace(8),
+              const RotatedBox(
+                quarterTurns: 0,
+                child: FlowySvg(
+                  FlowySvgs.arrow_tight_s,
+                  // size: Size.square(10),
+                ),
+              ),
```

Comment:
I think the RotatedBox is redundant here..

Reply:
Thanks, I screwed up while switching to the new svg haha